File: GTTS/INL.SearchService/Models/GetStudents_Item.cs
using System;

namespace INL.SearchService.Models
{
    public class GetStudents_Item : IGetStudents_Item
    {
        public long PersonID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        
        
        
        public DateTime? DOB { get; set; }
        public char Gender { get; set; }
        public string JobTitle { get; set; }
        public int? JobRank { get; set; }
        public int CountryID { get; set; }
        public string CountryName { get; set; }
        public string CountryFullName { get; set; }
        public string UnitName { get; set; }
        public string UnitNameEnglish { get; set; }
        public long? UnitMainAgencyID { get; set; }
        public string VettingStatus { get; set; }
        public DateTime? VettingStatusDate { get; set; }
        public string VettingType { get; set; }
        public string Distinction { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/get-students_item.ts


import { IGetStudents_Item } from './iget-students_item';

export class GetStudents_Item implements IGetStudents_Item {
  
	public PersonID: number = 0;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOB?: Date;
	public Gender: string = "";
	public JobTitle: string = "";
	public JobRank?: number;
	public CountryID: number = 0;
	public CountryName: string = "";
	public CountryFullName: string = "";
	public UnitName: string = "";
	public UnitNameEnglish: string = "";
	public UnitMainAgencyID?: number;
	public VettingStatus: string = "";
	public VettingStatusDate?: Date;
	public VettingType: string = "";
	public Distinction: string = "";
  
}


